### PR TITLE
AccountTokenRecord: Add wallet relation

### PR DESCRIPTION
**Pull request by Riddlerrr**
We decided to link AccountTokenRecord with wallet instead of Account because we will have the ability to have multiple wallets for blockchain. This change needs to be before we will continue with https://github.com/CoMakery/comakery-server/pull/1804

Also I've added `WalletTransferRule` as an alias for `AccountTokenRecord`. It's preparing for the task: https://www.pivotaltracker.com/story/show/176374185

### CODE REVIEW

**Comment by aquabu** [comment]
Note: this should be merged after the create multiple wallet PR.

File: db/data_migrations/20210111085831_fill_wallet_for_account_token_record.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class FillWalletForAccountTokenRecord < ActiveRecord::DataMigration
+  def up
+    AccountTokenRecord.where(wallet: nil).includes(:token, :reg_group).find_each do |atr|
```
Comment (aquabu) [inline db/data_migrations/20210111085831_fill_wallet_for_account_token_record.rb:3]: Could you write a test for this?

**Review by aquabu — CHANGES_REQUESTED** [review]

File: db/data_migrations/20210111085831_fill_wallet_for_account_token_record.rb (hunk 1 above)
Comment (Riddlerrr) [reply db/data_migrations/20210111085831_fill_wallet_for_account_token_record.rb:3]: 🤔 I think we don't have tools to test data migrations
@Alexey1100 Do you have any ideas about how we can add specs the data migration?

Comment (Riddlerrr) [reply db/data_migrations/20210111085831_fill_wallet_for_account_token_record.rb:3]: Added tests

**Review by Alexey1100 — APPROVED** [review]

**Review by aquabu — APPROVED** [review]

**Review by aquabu — APPROVED** [review]

**Pull request merged**